voltdbclient/test_verification.go

```diff
@@ -0,0 +1,82 @@
+/* This file is part of VoltDB.
+ * Copyright (C) 2008-2016 VoltDB Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with VoltDB.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package voltdbclient
+
+import (
+	"bytes"
+	"testing"
```

Comment:
This file seems a test helper file. not a source file, right?
Can we rename this to verification_test or move the logic to client_write_test ?
Otherwise other packages import voltdbclient will contain the side effect of "testing"